pkg/daemon/daemon.go

```diff
@@ -142,6 +144,22 @@ func NewUnikDaemon(config config.DaemonConfig) (*UnikDaemon, error) {
 		break
 	}
 
+	for _, openstackConfig := range config.Providers.Openstack {
+		logrus.Infof("Bootstrapping provider %s with config %v", openstack_provider, openstackConfig)
```

Comment:
Is here where you mentioned that entire OpenStack config is printed? Do you think it would be possible to mask the pass? I see that none of the other providers do it, but still.